composer.json

```diff
@@ -52,6 +52,7 @@
         "symfony/twig-bridge": "^2.8 || ^3.2 || ^4.0",
         "symfony/twig-bundle": "^2.8 || ^3.2 || ^4.0",
         "symfony/validator": "^2.8 || ^3.2 || ^4.0",
+        "symfony/webpack-encore-bundle": "^1.1",
```

Comment:
Shall we require `^1.6` here?

https://github.com/symfony/webpack-encore-bundle/releases

Reply:
We could yea. 👍